### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_Keyboard:Keyboard Focus is not visible on the cells present on the page while navigating through the "TAB" key.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4201]
Environment Details:
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 70.0.3538.77 (Official Build) (64-bit)Operating System:Windows10

Steps to reproduce:
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4.Navigate to code section on the page.

 
Actual
Keyboard Focus is not visible on the cells present on the page while navigating through the "TAB" key.
 
Expected
Keyboard Focus should be  visible on the cells present on the page while navigating through the "TAB" key.

User Impact
People who are blind (who cannot use devices such as mice that require eye-hand coordination)
People with low vision (who may have trouble finding or tracking a pointer indicator on screen)
Some people with hand tremors find using a mouse very difficult and therefore usually use a keyboard 

Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}}

[A11y_AzureNotebooks_Jupyter_CreationAndDeployment_Keyboard.zip](https://github.com/jupyter/notebook/files/2576073/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_Keyboard.zip)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
